native_client/dotnet/DeepSpeechClient/DeepSpeech.cs

```diff
@@ -229,9 +301,11 @@ public unsafe string SpeechToText(short[] aBuffer, uint aBufferSize)
         /// </summary>
         /// <param name="aBuffer">A 16-bit, mono raw audio signal at the appropriate sample rate (matching what the model was trained on).</param>
         /// <param name="aBufferSize">The number of samples in the audio signal.</param>
-        /// <returns>The extended metadata. The user is responsible for freeing the struct.  Returns NULL on error.</returns>
```

Comment:
Removed the "The user is responsible for freeing the struct" we already do it while marshaling.